tvtk/tests/test_vtk_parser.py

```diff
@@ -206,7 +226,7 @@ def test_method_signature(self):
                              p.get_method_signature(o.GetColor))
         if hasattr(vtk, 'vtkArrayCoordinates'):
             self.assertEqual([([None], ('float', 'float', 'float')),
-                ([None], (['float', 'float', 'float'],))],
+                              ([None], (['float', 'float', 'float'],))],
```

Comment:
tangential : after we do the release, i would really like to run black on the codebase to address all such indentation/style issues in one go.

Reply:
Sure, although I do not like black, maybe blue?